### PR TITLE
Persists Ids

### PR DESCRIPTION
**Pull request by jordanwa1947**
This pull request:
* persists the id count even after page load
* Removes window from localStorage calls because it's not necessary

### CODE REVIEW

**Review by ERaith — APPROVED** [review]
Looks good, takes out the global reference to the window object. I approve!

File: main.js [hunk 1]
```diff
@@ -84,8 +83,7 @@ function enableSaveButton() {
 
 function saveOutfit() {
   var curId = parseInt(localStorage.getItem('id'));
-  currentOutfit.id = curId;
-  localStorage.setItem('id', ++curId);
+  currentOutfit.id = new Date().valueOf();
```
Comment (jonathan-tschida) [inline main.js]: since we are using this id as an html id as well, it needs to start with a letter

Comment (jordanwa1947) [reply main.js]: Why is that?

Comment (jordanwa1947) [reply main.js]: Done

**Pull request merged**